### PR TITLE
feat(fx-2466/fx-2571): medium/classifications modal

### PR DESCRIPTION
**Pull request by dzucconi**
Closes: [FX-2466](https://artsyproduct.atlassian.net/browse/FX-2466)
Closes: [FX-2571](https://artsyproduct.atlassian.net/browse/FX-2571)

This uses the default behavior of the mobile modal

### CODE REVIEW

File: src/v2/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAdditionalInfo.tsx [hunk 1]
```diff
@@ -8,101 +9,123 @@ import {
...
-      { title: "Publisher", value: publisher },
-      { title: "Manufacturer", value: manufacturer },
-      { title: "Image rights", value: image_rights },
-    ]
+  const listItems = [
+    {
+      title: "Medium",
+      value: (
+        <>
+          {artwork.mediumType ? (
```
Comment (dzucconi) [inline src/v2/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAdditionalInfo.tsx:46]: It looks like there might be a bug in the resolver for `mediumType` as sometimes we have a `category` but no `mediumType` to correspond.

Comment (dblandin) [reply src/v2/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAdditionalInfo.tsx:46]: Let me know if you come across any specific examples. Might be due to a mismatch between the `category` value and the keys in https://github.com/artsy/metaphysics/blob/master/src/lib/artworkMediums.ts

**Review by dblandin — APPROVED** [review]
LGTM 👍

**Comment by dblandin** [comment]
Assigned back to you to merge when CI is 🍏

File: src/v2/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAdditionalInfo.tsx (hunk 1 above)
Comment (dzucconi) [reply src/v2/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAdditionalInfo.tsx:46]: The issue seems to be fixed now. May have just been dependent on PR from earlier hitting staging. 🤷  I was previously seeing `Drawing, Collage or other Work on Paper` with this, but looks good now.

Comment (dblandin) [reply src/v2/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAdditionalInfo.tsx:46]: Yep, that case was fixed with https://github.com/artsy/metaphysics/commit/9a0c5b71ae074a4f2add034c80ce56abc2dc5f5f

I only saw that one mismatch between https://github.com/artsy/kinetic/blob/3d997e714f55cce23a6432bffe01d04d2327fbd1/app/models/kinetic/artwork.rb#L231-L251 and https://github.com/artsy/metaphysics/blob/master/src/lib/artworkMediums.ts so hopefully we're covered now!

**Pull request merged**